f/export/postgres_to_file/tests/postgres_to_csv_test.py

```diff
@@ -0,0 +1,31 @@
+from f.export.postgres_to_file.postgres_to_csv import main
+
+
+def test_script_e2e(pg_database, database_mock_data, tmp_path):
+    asset_storage = tmp_path / "datalake/export"
+
+    main(
+        pg_database,
+        "comapeo_data",
+        asset_storage,
+    )
+
+    with open(asset_storage / "comapeo_data.csv") as f:
+        data = f.read()
+        assert data == (
```

Comment:
This is fine, but also FYI consider a "HEREDOC" to store the expected value.

https://stackoverflow.com/questions/9589301/python-here-document-without-newlines-at-top-and-bottom